a.js

```diff
@@ -0,0 +1,3 @@
+var bod_neme;
```

Comment:
format: typos:
Possible typos in `bod_neme`, suggestions:
Candidate #1 with a confidence 39%
```suggestion
var body_name;
```
Candidate #2 with a confidence 31%
```suggestion
var box_name;
```
Candidate #3 with a confidence 29%
```suggestion
var mod_name;
```